packages/rich-text/src/get-active-formats.js

```diff
@@ -1,9 +1,13 @@
 /**
  * Gets the all format objects at the start of the selection.
  *
- * @param {Object} value                Value to inspect.
- * @param {Array}  EMPTY_ACTIVE_FORMATS Array to return if there are no active
- *                                      formats.
+ * @param {Object}        value                Value to inspect.
```

Comment:
Same as your other PR, it would be good to create a type for a rich text value, then use it an all the other documentation. Might be good to also combine all rich text documentation updates into one PR.

Reply:
See https://jsdoc.app/tags-typedef.html for type definitions. This would definitely be useful as this rich text value types is used numerous times.

Reply:
This PR fixes existing warnings, can we merge it as is? The issue you mentioned existed before.

Reply:
This PR fixed the existing warning issue. But @ellatrix  tells something about `typedef`.